src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/SwitchPartitionsInAllTables.sql

```diff
@@ -1,7 +1,6 @@
 ﻿--IF object_id('SwitchPartitionsInAllTables') IS NOT NULL DROP PROCEDURE dbo.SwitchPartitionsInAllTables
 GO
 CREATE PROCEDURE dbo.SwitchPartitionsInAllTables
-WITH EXECUTE AS SELF
```

Comment:
Why is dbo not needed here?

Reply:
Permissions from one stored proc to another are transferable. I don't think impersonation is needed.